fuel/src/test/kotlin/com/github/kittinunf/fuel/RequestTest.kt

```diff
@@ -735,10 +741,10 @@ class RequestTest : BaseTestCase() {
         request.responseString().third.fold({ string ->
             try {
                 val json = JSONObject(string)
-                assertEquals(json.getJSONObject("args").getString("bar"),lionel)
+                assertEquals(json.getJSONObject("args").getString("bar"), lionel)
                 assertEquals(json.getJSONObject("args").getJSONArray("foo[]").map { it.toString() },
-                list.toList())
-            } catch (e: Exception){
+                        list.toList())
```

Comment:
[Super nits] Can we combine this with the previous line?

Reply:
done